app/src/main/java/fr/free/nrw/commons/upload/DescriptionsAdapter.java

```diff
@@ -122,7 +125,20 @@ public void init(int position) {
                 });
 
             } else {
-                descItemEditText.setCompoundDrawablesWithIntrinsicBounds(null, null, null, null);
+                descItemEditText.setCompoundDrawablesWithIntrinsicBounds(null, null,
```

Comment:
this is using the wrong `setCompoundDraawablesWithIntrinsicBounds` use the one that takes ints and you don't need this `getIcon` malarkey, just pass `0` instead of null